numba_cuda/numba/cuda/dispatcher.py

```diff
@@ -183,7 +184,7 @@ def __init__(
             self.argtypes,
             debug=self.debug,
             lineinfo=lineinfo,
-            inline=inline,
```

Comment:
Just double checking, since it is happening on LLVM IR level there is no need to pass `inline`, because it affects only Numba IR?

Reply:
Yeah, exactly - `inline` affects online Numba IR, `forceinline` affects only LLVM IR (which is how it should be when all is working correctly)